sdk/clientcore/ci.yml

```diff
@@ -50,6 +54,10 @@ extends:
         groupId: io.clientcore
         safeName: httpokhttp3
         releaseInBatch: ${{ parameters.release_clientcorehttpokhttp3 }}
+      - name: annotation-processor
+        groupId: io.clientcore
+        safeName: annotationprocessor
+        releaseInBatch: ${{ parameters.release_clientcoreannotationprocessor }}
     AdditionalModules:
```

Comment:
The annotation-processor-dependency-test package should be added to additional modules.